examples/run_generic_echo.py

```diff
@@ -131,8 +129,9 @@
     ["ph2", "ph_diff", "t2"],
     [len(ph2), len(ph_diff), -1],
 )
+data.set_units("t2", "s")
 data.setaxis("ph2", ph2 / 4).setaxis("ph_diff", ph_diff / 4)
-data.set_prop("postproc_type", "spincore_diffph_SE_v2")
+data.set_prop("postproc_type", "spincore_diffph_SE_v4")
```

Comment:
increment post processing version so that we aren't multiplying by nScans anymore